readme.md

```diff
@@ -1,26 +1,38 @@
 # babel-plugin-ember-test-metadata
 
-> A Babel plugin for Ember apps that adds test metadata
+> A Babel plugin for Ember apps that adds a test's file path to test metadata available through `@ember/test-helpers`.
 
 [![Package Version](https://img.shields.io/npm/v/ember-babel-add-test-metadata.svg?style=flat-square)](https://www.npmjs.com/package/ember-babel-add-test-metadata)
 [![Downloads Status](https://img.shields.io/npm/dm/ember-babel-add-test-metadata.svg?style=flat-square)](https://npm-stat.com/charts.html?package=ember-babel-add-test-metadata&from=2016-04-01)
-[![Build Status: Linux](https://img.shields.io/travis/glnster/ember-babel-add-test-metadata/master.svg?style=flat-square)](https://travis-ci.org/glnster/ember-babel-add-test-metadata)
-[![Coverage Status](https://img.shields.io/codecov/c/github/glnster/ember-babel-add-test-metadata/master.svg?style=flat-square)](https://codecov.io/gh/glnster/ember-babel-add-test-metadata)
+[![Code Style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](#badge)
 
-## Why
+A test file's file path can enable downstream consumers of test results to process, and potentially associate, a specific test failure to a specific file path. This can assist with analysis of test infrastructure, specifically where identifying ownership over a test file is useful or required.
```

Comment:
```suggestion
A test file's path can enable downstream consumers of test results to process, and potentially associate a specific test failure to a file path. This can assist with the analysis of test infrastructure, specifically where identifying ownership over a test file is useful or required.
```